bsdlib/doc/gnss_extension.rst

```diff
@@ -172,34 +172,47 @@ The error in position calculation, when compared to the actual position, can be
+     location.latitude          = latitude; /* Best estimate within maximum limit of 1800 km  */
+     location.longitude         = longitude;/* Best estimate within maximum limit of 1800 km */
+     location.altitude          = altitude; /* Actual altitude of the device in meters */
+     location.unc_semimajor     = 127;      /* Can be set to 127 or less if actual uncertainty is known */
+     location.unc_semiminor     = 127;      /* Can be set to 127 or less if actual uncertainty is known  */
+     location.orientation_major = 0;        /* Can be set to 0 if unc_semimajor and unc_semiminor are identical values */
+     location.unc_altitude      = 0;        /* Actual altitude, value needs to be less than 48 in order to be taken into use */
+     location.confidence        = 100;      /* Set to 100 for maximum confidence */
 
-.. code-block:: c
+ The struct contains the latitude, longitude (Geodetic latitude/longitude, WGS-84 format) and altitude (in meters) parameters.
+ The uncertainties for the coordinates (unc_semimajor and unc_semiminor) and for the altitude (unc_altitude) are given as an index from ``0`` to ``127``.
+ The altitude uncertainty must be less than 100 meters (index less than ``48``) for it to be valid as a reference altitude.
+ The accuracy of the latitude and longitude are less important, but it must be within 1800 kilometers of the actual location.
+
+If both verified GNSS fix (five or more satellites used in earlier fix) and A-GPS assistance data are available, then the altitude from the verified GNSS fix is used.
 
-   nrf_gnss_agps_data_location_t location;
+Thus, if GNSS has started in low accuracy mode, the GNSS engine will not be able to produce fixes using three satellites until it has a reference altitude from one of the above-mentioned sources.
+Furthermore, to continue having possible three satellite fixes, the reference altitude must be updated at least once in every 24 hours.
 
-   location.latitude          = 0;        /* Unknown, set to 0 */
-   location.longitude         = 0;        /* Unknown, set to 0 */
-   location.altitude          = altitude; /* Actual altitude of the device in meters */
-   location.unc_semimajor     = 127;      /* Set to 127 when location is unknown */
-   location.unc_semiminor     = 127;      /* Set to 127 when location is unknown */
-   location.orientation_major = 0;        /* Unknown, set to 0 */
-   location.unc_altitude      = 0;        /* Set to 0 */
-   location.confidence        = 100;      /* Set to 100 */
+.. note::
+   Starting or stopping the navigation with bit ``3`` (``delete the last valid fix``) in the delete mask clears the reference altitude value.
+
+
+.. important::
+   The altitude must be accurate to a value within ±100 meters of the actual altitude of the device. An erroneous altitude will result in a severe error in the position fix calculation using three satellites.
```

Comment:
This was probably changed to 100 meters because it is said above that altitude uncertainty must be less than 100 meters. These values are different on purpose. The given altitude should be accurate (+- 10 meters) for the position to be accurate when three satellites are used. Altitude uncertainty must be less than 100 meters (index less than 48), otherwise GNSS will not use the given altitude at all. But for accurate position fixes using three satellites, the given altitude should be much more accurate (+- 10 meters).

Reply:
Updated to ±10 meters